test/membrane_http_adaptive_stream/integration_test/source_test.exs

```diff
@@ -4,22 +4,29 @@ defmodule Membrane.HTTPAdaptiveStream.Source.Test do
   import Membrane.ChildrenSpec
   import Membrane.Testing.Assertions
 
-  alias Membrane.{AAC, H264, RemoteStream}
+  require Logger
 
+  alias Membrane.{AAC, H264, RemoteStream}
   alias Membrane.Testing
 
   @mpegts_url "https://test-streams.mux.dev/x36xhzz/x36xhzz.m3u8"
   @fmp4_url "https://raw.githubusercontent.com/membraneframework-labs/ex_hls/refs/heads/plug-demuxing-engine-into-client/fixture/output.m3u8"
+  @bbb_33s_mp4_url "https://github.com/membraneframework/static/raw/refs/heads/gh-pages/samples/big-buck-bunny/bun33s.mp4"
```

Comment:
I think permlink looks more like this: `https://github.com/membraneframework/static/blob/4fb2cbe290c4ee2628ca4fa3997b6402fb05a9a8/samples/big-buck-bunny/bun33s.mp4` (if we left a reference to gh-pages, we risk that the tests would start failing after new commits to the `static` repo)